frontend/desktop/data/transform.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+
+var readFileSync = require('fs').readFileSync;
+var writeFileSync = require('fs').writeFileSync;
+var yaml = require('js-yaml');
+var _ = require('lodash');
+
+if (process.env.NODE_ENV === 'development') {
+  try {
+    readFileSync('./data/config.local.yaml', 'utf-8');
+  } catch (e) {
+    if (e.code === 'ENOENT') {
+      writeFileSync('./data/config.local.yaml', readFileSync('./data/config.yaml', 'utf-8'));
+      console.log('Copied ./data/config.yaml to ./data/config.local.yaml');
+    } else {
+      throw e;
+    }
+  }
+}
+
+let yamlObj;
+const filename =
+  process.env.NODE_ENV === 'development' ? './data/config.local.yaml' : '/app/data/config.yaml';
```

Comment:
During the build process, if this configuration file is missing, problems will occur.